fluss-server/src/main/java/com/alibaba/fluss/server/tablet/TabletServer.java

```diff
@@ -112,7 +112,7 @@ public class TabletServer extends ServerBase {
     private ReplicaManager replicaManager;
 
     @GuardedBy("lock")
-    private @Nullable RemoteLogManager remoteLogManager = null;
+    private final @Nullable RemoteLogManager remoteLogManager = null;
```

Comment:
Instead of making `remoteLogManager` to be `final`, I think we need to assign `remoteLogManager` with `remoteLogManager = replicaManager.getRemoteLogManager()` after the code line `replicaManager.startup();` so that we can close `remoteLogManager` after tablet server is close.
May @swuferhong help check this ?

Reply:
OK, I have already keep it as origin now.